.github/workflows/lambda-ci.yml

```diff
@@ -0,0 +1,35 @@
+name: Lambda CI - ZIP 
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'backend/lambda_functions/**'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4.1.5
+
+      - name: Build zip files for each lambda function
+        run: |
+          cd backend/lambda_functions
+          for d in */ ; do
+            zip -r "${d%/}.zip" "$d"
+          done
+
+      # - name: Configure AWS Credentials
```

Comment:
I understand why this is commented out at the moment, but in next few days, we'll need to use role assumption instead of this less secured access-key token.

Reply:
Got it :)